# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/script/moz.build
# Commit: 0b3f0972e002
# Full Hash: 0b3f0972e002f6b010b4683fc7c8e4e366b2cf4e
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-21 21:39:46
# Regressor Bug: 1742437
# File Overlap Count: 1
# Description:
#   Bug 1742437 - move DOM module loader into own file; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D137531
# ==============================================================================

diff -r 98d6cb49f4fd -r 0b3f0972e002 dom/script/moz.build
--- a/dom/script/moz.build	Mon Feb 21 15:39:07 2022 +0000
+++ b/dom/script/moz.build	Mon Feb 21 15:39:08 2022 +0000
@@ -20,6 +20,7 @@
 EXPORTS.mozilla.dom += [
     "AutoEntryScript.h",
     "LoadedScript.h",
+    "ModuleLoader.h",
     "ModuleLoaderBase.h",
     "ModuleMapKey.h",
     "ScriptDecoding.h",
@@ -33,6 +34,7 @@
 UNIFIED_SOURCES += [
     "AutoEntryScript.cpp",
     "LoadedScript.cpp",
+    "ModuleLoader.cpp",
     "ModuleLoaderBase.cpp",
     "ModuleLoadRequest.cpp",
     "ModuleMapKey.cpp",
